STUDIO/org.openl.rules.webstudio/src/org/openl/rules/rest/UsersController.java

```diff
@@ -1,13 +1,5 @@
 package org.openl.rules.rest;
 
-import static org.openl.rules.ui.WebStudio.TABLE_FORMULAS_SHOW;
-import static org.openl.rules.ui.WebStudio.TABLE_VIEW;
-import static org.openl.rules.ui.WebStudio.TEST_FAILURES_ONLY;
-import static org.openl.rules.ui.WebStudio.TEST_FAILURES_PERTEST;
-import static org.openl.rules.ui.WebStudio.TEST_RESULT_COMPLEX_SHOW;
-import static org.openl.rules.ui.WebStudio.TEST_TESTS_PERPAGE;
-import static org.openl.rules.ui.WebStudio.TRACE_REALNUMBERS_SHOW;
-
```

Comment:
Don't override the default OpenL code style. Please, install Eclipse Code Formatter Plugin and set it to https://github.com/openl-tablets/openl-tablets/blob/master/OpenL_Code_Style.xml